demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -1395,6 +1435,11 @@ export class DemoMeetingApp implements AudioVideoObserver, DeviceChangeObserver,
     this.log(`session stopped from ${JSON.stringify(sessionStatus)}`);
     if (sessionStatus.statusCode() === MeetingSessionStatusCode.AudioCallEnded) {
       this.log(`meeting ended`);
+      // @ts-ignore
+      window.location = window.location.pathname;
```

Comment:
Oh i see, we are performing the redirection only after the 'AudioCallEnded' and not after the leave / end button is clicked.